src/handlers/get.ts

```diff
@@ -33,6 +33,12 @@ const getHandler: Handler = async (request, env, ctx) => {
     return BAD_REQUEST;
   }
 
+  if (requestUrl.pathname === '/robots.txt') {
```

Comment:
I also wonder if this should only be applied to r2.nodejs.org or all domains? cc @targos

Reply:
The domains this would apply on would only be `r2.nodejs.org`, `dist-worker-staging.nodejs.workers.dev`, and `dist-worker-prod.nodejs.workers.dev`